src/NzbDrone.Core/NetImport/NetImportDefinition.cs

```diff
@@ -7,6 +8,11 @@ namespace NzbDrone.Core.NetImport
 {
     public class NetImportDefinition : ProviderDefinition
     {
+        public NetImportDefinition()
+        {
+            Tags = new HashSet<int>();
+        }
```

Comment:
Would it make sense to add Tags to providerBase instead of each provider type? This is what Sonarr did in V3.

Reply:
@Qstick Can you clarify what you mean? I mean, I'm **only** adding this to `NetImportDefinition` not "each provider type". Also, are you referring to `ProviderDefinition` by "providerBase"? But then again, I'm not sure I understand what you are saying.

Reply:
@Qstick Can you clarify what you mean? I mean, I'm only adding this to NetImportDefinition not "each provider type". Also, are you referring to ProviderDefinition by "providerBase"? But then again, I'm not sure I understand what you are saying.

Reply:
Yea I’d say this is fine for now given tags are not used for any other provider type, in sonarr tags are also used for other thingi provider types, thus the tags property was moved to the base definition in v3. That’s all I meant.